work/service/work_handlers.go

```diff
@@ -0,0 +1,85 @@
+package service
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	dataService "github.com/tidepool-org/platform/data/service"
+	request "github.com/tidepool-org/platform/request"
+	workLoad "github.com/tidepool-org/platform/work/load"
+)
+
+func LoadTestRoutes() []dataService.Route {
+	return []dataService.Route{
+		dataService.Get("/v1/work/load/ok", okHandler),
+		dataService.Post("/v1/work/load", createHandler),
+		dataService.Delete("/v1/work/load/:groupId", deleteHandler),
+	}
+}
+
+func okHandler(dataServiceContext dataService.Context) {
+	res := dataServiceContext.Response()
+	req := dataServiceContext.Request()
+	responder := request.MustNewResponder(res, req)
+	responder.Data(http.StatusOK, map[string]any{"status": "All good ready for work load"})
+}
+
+func deleteHandler(dataServiceContext dataService.Context) {
+	ctx := dataServiceContext.Request().Context()
+	res := dataServiceContext.Response()
+	req := dataServiceContext.Request()
+
+	responder := request.MustNewResponder(res, req)
+
+	groupID := req.PathParam("groupId")
+	if groupID == "" {
+		responder.Error(http.StatusBadRequest, request.ErrorParameterMissing("groupId"))
+		return
+	}
+
+	_, err := dataServiceContext.WorkClient().DeleteAllByGroupID(ctx, groupID)
+
+	if err != nil {
+		responder.Error(http.StatusInternalServerError, err)
+		return
+	}
+
+	responder.Empty(http.StatusNoContent)
```

Comment:
200 is more typical for a delete operation. 204 is more for a get where this isn't anything to get.